core/src/text.rs

```diff
@@ -40,6 +40,23 @@ pub struct Text<'a, Font> {
 
     /// The [`Shaping`] strategy of the [`Text`].
     pub shaping: Shaping,
+
+    /// The [`Wrapping`] strategy of the [`Text`].
+    pub wrapping: Wrapping,
+}
+
+/// The wrapping strategy of some text.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Default)]
+pub enum Wrapping {
+    /// No wrapping.
+    ///
+    /// This is the default.
+    None,
+    /// Word wrapping.
+    #[default]
+    Word,
```

Comment:
It seems that default is `Word` if see at https://github.com/iced-rs/iced/pull/2279/files#diff-b8dc21f0859d1f4cc40ec8a7bc71112427a373b082e7d101a94fe807887b54b4R54

Reply:
The documentation needs to be updated.

Reply:
Updated